docs/book/src/cronjob-tutorial/testdata/project/config/default/kustomization.yaml

```diff
@@ -36,7 +36,7 @@ patchesStrategicMerge:
 
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
 # crd/kustomization.yaml
-#- manager_webhook_patch.yaml
+- manager_webhook_patch.yaml
```

Comment:
That is a very good catcher but we need to fix it in the hack/ files which generate the sample mocks
Otherwise, it will not pass in the CI since this change will be overwriting when we run `make generate` and it calls the make generate-docs to automatically update the samples under docs with the latest changes

Reply:
Done